src/services/ColorService.js

```diff
@@ -5,4 +5,5 @@ const speciesColorMap = Map({'black bear':'#801e78','bobcat': '#498029','coyote'
 export const getColorForSpecies = (species) => {
     const color = speciesColorMap.get(species.toLowerCase());
     return color ? color : '#805b14';
-}
+};
+export {speciesColorMap};
```

Comment:
I'm not a huge fan of exporting the underlying data structure itself -- I'd rather export functions that give us the information we need, and handle any parsing in this class itself.